packages/core/src/provider/base.ts

```diff
@@ -352,15 +379,13 @@ export class SmartAccountProvider<
   // You should implement your own middleware to override these
   // or extend this class and provider your own implemenation
   readonly dummyPaymasterDataMiddleware: AccountMiddlewareFn = async (
-    struct: UserOperationStruct
-  ): Promise<UserOperationStruct> => {
+    struct
```

Comment:
nit: we don't want to type the inputs?

Reply:
it's already typed because of the interface that this class implements